db/view/view_update_backlog.hh

```diff
@@ -28,28 +29,11 @@ struct update_backlog {
         return float(current) / float(max);
     }
 
-    friend bool operator==(const update_backlog& lhs, const update_backlog& rhs) {
-        return lhs.relative_size() == rhs.relative_size();
+    std::partial_ordering operator<=>(const update_backlog &rhs) const {
```

Comment:
partial_ordering makes sense here, because a == b doesn't mean a and b are interchangable.